interfaces/Glitter/templates/static/javascripts/glitter.main.js

```diff
@@ -388,6 +414,23 @@ function ViewModel() {
             self.isRestarting(1)
         }).always(self.setNextUpdate);
 
+        // this is a test of getting the entire queue
```

Comment:
:welp: looks like it does muck with the priority for some reason.  if slot 0 is a high priority and an item is moved up to slot 0 it gets high priority and keeps it.   That has to be some API mumbo jumbo but does look like we need to know the full queue to even begin to figure out here to go if we need to take priority into account.  @Safihre i have a commit to remove the "test" verbiage but can hold off on it and refocus on the behavior of the move if you want that included in this pull..